spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -2,12 +2,12 @@
 require "rails_helper"
 
 RSpec.describe Admin::CategoriesController, type: :controller do
-  describe "GET index" do
-    before do
-      @admin = FactoryBot.create(:user)
-      @user = FactoryBot.create(:user, role: 0)
-    end
+  before do
+    @admin = FactoryBot.create(:user)
+    @user = FactoryBot.create(:user, role: 0)
```

Comment:
tạo biến này mà em ko dùng nhỉ?

Reply:
`it "return a 302 response" do
          sign_in @user`
e có xài nha c, line 39